spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -253,21 +253,21 @@
         },
         {
           "title" => "Invalid study mode",
-          "detail" => "You need to pick an option",
+          "detail" => "Select a study mode",
           "source" => {
             "pointer" => "/data/attributes/study_mode",
           },
         },
         {
           "title" => "Invalid age range in years",
-          "detail" => "You need to pick an age range",
+          "detail" => "Select an age range",
           "source" => {
             "pointer" => "/data/attributes/age_range_in_years",
           },
         },
         {
           "title" => "Invalid level",
-          "detail" => "You need to pick a level",
+          "detail" => "Select a level",
```

Comment:
Change to 'Select a course level' (if this relates to Primary/Secondary/Further education).